plugins/registry/functions.go

```diff
@@ -100,9 +108,29 @@ func pushToRegistry(appName string, tag int, imageID string, imageRepo string) e
 		return errors.New("Unable to tag image")
 	}
 
-	// For the future, we should also add the ability to create the remote repository
-	// This is only really important for registries that do not support creation on push
-	// Examples include AWS and Quay.io
+	extraTags := getRegistryPushExtraTagsForApp(appName)
+	if extraTags != "" {
+		extraTagsArray := strings.Split(extraTags, ",")
+		for _, extraTag := range extraTagsArray {
+			extraTagImage := fmt.Sprintf("%s%s:%s", registryServer, imageRepo, extraTag)
+			common.LogVerboseQuiet(fmt.Sprintf("Tagging %s as %s in registry format", imageRepo, extraTag))
+			if !dockerTag(imageID, extraTagImage) {
+				return errors.New(fmt.Sprintf("Unable to tag image as %s", extraTag))
+			}
+			func() {
+				defer func() {
+					common.LogVerboseQuiet(fmt.Sprintf("Untagging extra tag %s", extraTag))
+					if !dockerRemoveTag(extraTagImage) {
```

Comment:
Please use `common.RemoveImages` instead:

```go
if err := common.RemoveImages(extraImageTag); err != nil {
    common.LogWarn(fmt.Sprintf("Unable to remove extra tag '%s': %s", extraTag, err.Error()))
}
```

And then remove the `dockerRemoveTag` func.

Reply:
removed, please have a look